### PR TITLE
feat(stdlib): use @logdna/stdlib for utilities

### PR DESCRIPTION
**Pull request by smusali**
feat(stdlib): use @logdna/stdlib for utilities

SemVer: patch

### CODE REVIEW

File: lib/utils/check-file-format.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+'use strict'
+
+module.exports = function checkFileFormat(key) {
+  if (!key || typeof key !== 'string') return
+
+  return {
+    json: key.indexOf('.json') >= 0 && key.indexOf('.jsonl') === -1
+  , gz: key.indexOf('.gz') >= 0
```
Comment (david-ly) [inline lib/utils/check-file-format.js]: Nit: I think using `key.includes('...')` instead of `indexOf` makes the intention here a bit more clear

**Review by david-ly — DISMISSED** [review]
lgtm only had 1 non-blocking comment

File: lib/utils/check-file-format.js (hunk 1 above)
Comment (smusali) [reply lib/utils/check-file-format.js]: Done!

**Comment by smusali** [comment]
@david-ly @esatterwhite @darinspivey @mdeltito 

Can you guys review this? :) Thanks!

**Review by mdeltito — APPROVED** [review]

**Review by david-ly — APPROVED** [review]

**Pull request merged**